internal/cmd/hana/check.go

```diff
@@ -143,10 +143,10 @@ func handleCheckResponse(u *unstructured.Unstructured, err error, printedName, n
 	return nil
 }
 
-func getServiceInstance(config *hanaCheckConfig) (*unstructured.Unstructured, error) {
+func getServiceInstance(config *hanaCheckConfig, name string) (*unstructured.Unstructured, error) {
```

Comment:
I see incostistency with params. IMO we should pass `(config)` or `(ctx, name)`. Now we have struct and next argument from this struct.